contrib/test/integration/swap.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Obtain current state of swap
+  command: swapon --noheadings --show=NAME
+  register: swapon
+
+- name: Setup swap if none already, to prevent kernel firing off the OOM killer
+  block:
+
+    - name: A unique swapfile path is generated
+      command: mktemp --tmpdir=/root swapfile_XXX
+      register: swapfilepath
+
+    - name: Swap file path is buffered
+      set_fact:
+        swapfilepath: '{{ swapfilepath.stdout | trim }}'
+
+    - name: Set swap file permissions
+      file:
+        path: "{{ swapfilepath }}"
+        owner: root
+        group: root
+        mode: 0600
+
+    - name: Swapfile is padded to 8-gig and formatted
+      shell: 'time dd if=/dev/zero of={{ swapfilepath }} bs=8M count=1000 && mkswap {{ swapfilepath }}'
```

Comment:
Did you really mean to leave the `time` in there? And, is there a reason for the `&&` instead of breaking the `mkswap` out into a separate step? Otherwise LGTM.

Reply:
No I didn't, thanks for catching that.  Though now that I think about it, maybe it's better left in there.  If there's ever a spike in test-times, having that statistic will be helpful for diagnosis,  e.g. sometimes VMs are backed by magnetic drives (which could take minutes to write all those zeros).

Only reason I put both commands in one, is to reduce the consumption of vertical space.

Reply:
Okay if I leave this as is...since running/passing all the tests again is a giant time-sink?

Reply:
OK with me. Merge away!

Reply:
The hard codes 8m here worries me a bit.  Would love to know if it makes @mrunalp twitch at all.

Reply:
FWIW: I at least thought about putting that into ``vars.yml``...briefly 😀

Happy to revisit that idea if it's needed.